HARK/ConsumptionSaving/tests/test_ConsAggShockModel.py

```diff
@@ -185,15 +189,15 @@ def test_methods(self):
 
         # testing update_solution_terminal()
         self.assertEqual(
-            self.agent.solution_terminal.cFunc[0](10,self.economy.MSS),
+            self.agent.solution_terminal.cFunc[0](10, 13.32722),
```

Comment:
This test checks a solution, not a simulation. I'd leave it in; there is nothing random about it.

Reply:
What is `economy.MSS`?

Reply:
Oh, it's `self.kSS * self.RfreeSS + self.wRteSS`. Of course. I'll change it back.

Reply:
Actually, MSS is stochastic, so this test and the one above can leave.